src/pages/pokedex-site.tsx

```diff
@@ -24,36 +67,77 @@ export default function PokedexSite() {
 
     return (
         <Layout>
-            <p className="dex-header-text">
-                Show Pokemon with the following type:
-            </p>
-            <button
-                className="button reset"
-                onClick={() => {
-                    setPokemonType(undefined);
-                }}
-            >
-                Reset Filter
-            </button>
-            <div className="button-wrapper">
-                {buttonTypes.map((buttonType) => {
-                    return (
-                        <button
-                            key={buttonType}
-                            className={`button ${buttonType} filter-button `}
-                            onClick={() => {
-                                setPokemonType(buttonType);
-                            }}
-                        >
-                            {buttonType}
-                        </button>
-                    );
-                })}
-            </div>
-            <div id="pokemon-card-wrapper">
-                {filteredList.map((pokemon) => {
-                    return <PokemonCard key={pokemon.name} pokemon={pokemon} />;
-                })}
+            <div className="grid grid-cols-3 gap-2">
+                <div className="flex flex-col gap-2 w-3/4">
```

Comment:
`w-3/4` ist ein seltsamer Wert in diesem Fall, und ist sicherlich am Desktop Viewport bemessen. Funktioniert das auch in kleineren Auflösungen? Vermutlich nicht.

Überlege dir, wie breit das Element eigentlich sein soll und definiere eine sinnigere Breite.